bugbot/rules/component.py

```diff
@@ -89,6 +89,21 @@ def get_bugs(self, date="today", bug_ids=[]):
         # Classify those bugs
         bugs = get_bug_ids_classification("component", bug_ids)
 
+        # Collect bugs classified as Fenix::General
+        fenix_general_bug_ids = [
+            bug_id
+            for bug_id, bug_data in bugs.items()
+            if "class" in bug_data and bug_data["class"] == "Fenix::General"
```

Comment:
We could simplify this by using `get()`:
```suggestion
            if bug_data.get("class") == "Fenix::General"
```

Reply:
Done in 246a34614bbe11b6aa3783164a10ce6336f20d09!